src/BoosterFactory.ts

```diff
@@ -2039,6 +2054,93 @@ class MKMBoosterFactory extends BoosterFactory {
 	}
 }
 
+// Outlaws of Thunder Junction
+// You can find a card from The List in 1 out of 5 Play Boosters. The List for OTJ has 40 cards: 30 cards from The Big Score and 10 Special Guests cards (SPG). You'll find a non-foil Special Guests card in 1 out of 64 Play Boosters.
+class OTJBoosterFactory extends BoosterFactory {
+	static readonly ListRatio: number = 1 / 5;
+	static readonly SPGRatio: number = 1 / 64;
+
+	otp: SlotedCardPool; // Breaking News
+	big: CardPool; // Big Score
+	spg: CardPool; // Special Guests
+
+	constructor(cardPool: SlotedCardPool, landSlot: BasicLandSlot | null, options: BoosterFactoryOptions) {
+		const opt = { ...options };
+		opt.foil = false; // We'll handle the garanteed foil slot ourselves.
+		super(cardPool, landSlot, opt);
+		this.otp = {};
+		for (const c of BoosterCardsBySet["otp"]) {
+			const rarity = getCard(c).rarity;
+			if (!this.otp[rarity]) this.otp[rarity] = new CardPool();
+			this.otp[rarity].set(c, options.maxDuplicates?.[rarity] ?? DefaultMaxDuplicates);
+		}
+		this.big = new CardPool();
+		for (const c of BoosterCardsBySet["big"])
+			this.big.set(c, options.maxDuplicates?.[getCard(c).rarity] ?? DefaultMaxDuplicates);
+		this.spg = new CardPool();
+		for (const c of SpecialGuests.otj)
+			this.spg.set(c, options.maxDuplicates?.[getCard(c).rarity] ?? DefaultMaxDuplicates);
+	}
+
+	generateBooster(targets: Targets) {
```

Comment:
Great work from trying to generate play boosters while all info is not public!

Unless I'm mistaken, I think there is a missing wildcard slot in the booster? (the non foil one third in this list https://magic.wizards.com/en/news/feature/collecting-outlaws-of-thunder-junction)
![image](https://github.com/Senryoku/Draftmancer/assets/34709007/9af96fdc-27c1-438f-8c45-d5ad3d3569f1)

No idea of how the probabilities for common/uncommon work on that one, but rare+mythics appear 1/12 of the time there, and an individual rare should probably appear twice as much as an individual mythic? Maybe?

Reply:
Mmh, yeah, good catch. 
I don't know why these posts confuse me so much 🙄

Reply:
They are very confusing (and don't get me started on the booster fun variants), and I'm sure that looks like they provide perfect pack composition to most people, but actually they lack some major information.